src/main/java/frc/robot/subsystems/vision/Limelight.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.net.PortForwarder;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import frc.robot.subsystems.LoggedSubsystem;
+
+import java.util.Optional;
+import java.util.OptionalDouble;
+
+public class Limelight extends LoggedSubsystem<LimelightLogInputs> {
+    public static Limelight INSTANCE = null;
+
+    private final NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+    private final NetworkTableEntry tx = table.getEntry("tx");
+    private final NetworkTableEntry ty = table.getEntry("ty");
+    private final NetworkTableEntry tv = table.getEntry("tv");
+    private final NetworkTableEntry ts = table.getEntry("ts");
+    private final NetworkTableEntry tid = table.getEntry("tid");
+    private LimelightLogInputs inputs = LimelightLogInputs.getInstance();
+
+    private Limelight() {
+        super(new LimelightLogInputs());
+        inputs = LimelightLogInputs.getInstance();
+        PortForwarder.add(5801, "limelight.local", 5801);
+    }
+
+    public Limelight getINSTANCE() {
```

Comment:
Method names in Java are written in `camelCase`.

Reply:
Why did you resolve this thread?

Reply:
I think katzuv means that according to the conventions of the team, the name of the function should be `getInstance()`